assets/js/src/forms/heading.jsx

```diff
@@ -5,35 +5,33 @@ import MailPoet from 'mailpoet';
 import { TopBarWithBeamer } from 'common/top_bar/top_bar';
 import plusIcon from 'common/button/icon/plus';
 
-const FormsHeading = ({ history }) => {
-  const goToSelectTemplate = () => {
-    setTimeout(() => {
-      window.location = window.mailpoet_form_template_selection_url;
-    }, 200); // leave some time for the event to track
-  };
-
-  return (
-    <>
-      <TopBarWithBeamer onLogoClick={() => history.push('/')}>
-        <Link
-          className="mailpoet-button"
-          to="/new"
-          onClick={goToSelectTemplate}
-          data-automation-id="create_new_form"
-        >
-          {plusIcon}
-          <span>{MailPoet.I18n.t('new')}</span>
-        </Link>
-      </TopBarWithBeamer>
-      <h1 className="mailpoet-newsletter-listing-heading-empty title">{' '}</h1>
-    </>
-  );
+export const goToSelectTemplate = () => {
+  setTimeout(() => {
```

Comment:
Do we need the timeout here? I don't see any events here mentioned in the comment, or should we add an event?

Reply:
Hmm, good question. I just moved the code. I can't see any event even before the change. I'll try to find out what happened to it.

Reply:
I **think** we used to have a Mixpanel event for tracking a form creation. That may have happened here.

Reply:
Hmm, the event is gone :( I'll add it to template selection function.